src/Ticket.php

```diff
@@ -4052,6 +4052,64 @@ public static function computeTco(CommonDBTM $item)
         return $totalcost;
     }
 
+    /**
+     * Return an array of predefined fields while also loading them into the $options array
+     *
+     * @param ITILTemplate $tt The ticket template to use
+     * @param array $options The current options array
```

Comment:
maybe add an indication here in comment to let developpers know `$options` is PASSED BY REFERENCE (in majuscule)